### ISSUE TITLE
Help people collect councillor information by providing them some instructions on where to get it

### ISSUE THREAD

**Issue by equivalentideas** [openaustralia/planningalerts#1182]
Collecting councillor information can be tricky if you don't know where to find it.

**Comment by hisayohorie** [comment]
I think this tutorial fro setlist.fm is a good example

http://www.setlist.fm/tutorial

**Comment by hisayohorie** [comment]
go to the council website. To be sure, enter the province name where the council exist
(ex. "Cessnock City Council, New South Wales”, "George Town Council, Tasmania”)
[embed google search bar]
make sure it open the new tab so people can see the instruction

Typically, There are a navigation menu says something like 

- About Council
- Council
- Your Council
- Our Council
- About us

<img width="1246" alt="screen shot 2017-06-26 at 7 30 39 pm" src="https://user-images.githubusercontent.com/12241881/27592539-b7e475be-5b22-11e7-84f9-01c6af0f1089.png">
<img width="1254" alt="screen shot 2017-06-27 at 10 28 11 am" src="https://user-images.githubusercontent.com/12241881/27592876-9ff8dc82-5b23-11e7-8c87-8aeae5d49aab.png">
<img width="1160" alt="screen shot 2017-06-27 at 10 52 27 am" src="https://user-images.githubusercontent.com/12241881/27594089-e6d6f7f8-5b26-11e7-9618-9479830eba0b.png">

You may have to click the tab, or sometime you have to put your cursor on the tab and the submenu show up. 
<img width="1162" alt="screen shot 2017-06-27 at 11 13 06 am" src="https://user-images.githubusercontent.com/12241881/27595135-d1243ae4-5b29-11e7-83fa-eef3bab544fd.png">

<img width="1280" alt="screen shot 2017-06-26 at 7 31 02 pm" src="https://user-images.githubusercontent.com/12241881/27594532-0508769c-5b28-11e7-8745-6f9f7269325a.png">
<img width="1280" alt="screen shot 2017-06-26 at 7 31 20 pm" src="https://user-images.githubusercontent.com/12241881/27594535-075f102c-5b28-11e7-81e3-de2999586c99.png">

 
And under the page, typically there are categories like
- Councillor’s profile
- Councillors
- Mayer and Councillors
- Councillors and Wards
- Ellected Councillors

<img width="1217" alt="screen shot 2017-06-26 at 8 07 35 pm" src="https://user-images.githubusercontent.com/12241881/27594962-4a8fb6b6-5b29-11e7-8d7e-98f245eb08fd.png">
<img width="1248" alt="screen shot 2017-06-27 at 10 25 45 am" src="https://user-images.githubusercontent.com/12241881/27594183-276dbcca-5b27-11e7-8e7c-65bc8edf5953.png">



[screen capture or short video/gif]

Different council has different navigation to the councillors information. Here are some example of the pass
City of Fremantle, West Australia
Home > Council > Your Council > Your Elected Members 
[screencapture]
Alpine Shire Council, Victoria
Home > Council > About Council > Councillors
[screencapture]
Carrathool Shire Council, New South Wales
Home > Council > Councillors
[screencapture]


Then finally, you will see a list of councillor information
[screen capture or short video/gif]

What we want are:
Name of the councillor 
Email address

Keep 
special characters (hyphens”-“, equal “=“)
Capitalization, but not ALL capitals( O Jane Green X jane green X Jane GREEN)

Leave
Title (Cr. Grant Henley -> Grant Henley, Dr. John Trembley -> John Trembley)


And sometime, it’s more complicated. If you already know the name of the councillor or the council you want to update, google
the name of the councillor
name of the council
name of the province 

[screen capture]

Check the spelling once again, if it looks fine, submit!
Your contribution will be live once it is approved. Thanks to you, soon people can write to this councillor in PlanningAlert!! Amazing!

**Comment by equivalentideas** [comment]
Looking good @hisayohorie what's the next step with this?

**Comment by hisayohorie** [comment]
@equivalentideas I think we should probably think about what are the parts we can automate by validation etc.

**Comment by equivalentideas** [comment]
@hisayohorie nice. I recon we could do some clever stuff with the name validation stuff at the end down the line.

Do you think it would be good to structure this in a series of steps like a recipe? I definitely find that kind of structure very helpful because it feels like you're making progress as you move through and it's easy to keep track of where you're at and where it fits.

Should these instructions go on the same page as the form or another page?

**Comment by hisayohorie** [comment]
@equivalentideas Agreed. The milestone structure has been helpful for me to keep tracking things. Probably we can extract and make another set of issues.

**Comment by hisayohorie** [comment]
Things we can do to make the instruction embedded in the contribution process

To make google search phase easier, we can
- Create a directory of Australian council names and their websites

To make the input process easier, we can
- Use validation such as, namecase, emailcase, titlecase, specifying a list of special characters to accept, and specifying the list of titles to exclude from saving.

Anything else? @equivalentideas @henare

**Comment by hisayohorie** [comment]
Directories of Australian local governments(links to the councils in the state)

- **New South Wales**
https://www.lgnsw.org.au/about-us/council-links

- **Western Australia**
http://walga.asn.au/About-Local-Government/Online-Local-Government-Directory.aspx

- **South Australia**
http://www.lga.sa.gov.au/councils

-** Tasamania**
http://www.lgat.tas.gov.au/page.aspx?u=225

-** Queensland**
http://www.bettercouncils.net.au/find-your-council

-** Victoria**
https://knowyourcouncil.vic.gov.au/councils

- ** Northern Territory**
http://www.lgant.asn.au/councils-2/

**Comment by hisayohorie** [comment]
Are there ways to extract the name of the authority and the council website from the csv file and convert them into json?

**Comment by henare** [comment]
> We can just add a website_urlcolumn in authority table

@hisayohorie I think what you're talking about here is documented in #442.

**Comment by equivalentideas** [comment]
Hey @hisayohorie it's great to see the tutorial in the PR. It would be really helpful if you've update the issue with what you're working on before starting the PR's so the connection is really clear (here's an example of the https://github.com/openaustralia/planningalerts/issues/1058#issuecomment-308942885).

I think we're jumping the gun a bit here with these two pull requests https://github.com/openaustralia/planningalerts/pull/1196 https://github.com/openaustralia/planningalerts/pull/1197 . @hisayohorie I take it these relate to your idea above:

> To make google search phase easier, we can:
>
> * Create a directory of Australian council names and their websites

I think you've gone down a more complex path than necessary at this stage by working on ways to make it easier to find the council, rather than just adding the basic instructions for starters. What do you recon? I feel like I'd like to see a PR for just adding the basic instructions before working on optimizing the process.

(update) It wasn't clear to me at first that a tutorial page was part of https://github.com/openaustralia/planningalerts/pull/1197 because the title was about something different. I still think we should look at the tutorial page as a separate issue from optimizing the process. I'll open up some new issues from the good ideas for optimising you posted above.

**Comment by equivalentideas** [comment]
To keep this issue focused on the simplest version of this, I've extracted the ideas for optimizations into new issues:

* Help people contribute correct councillor email addresses by validating their input #1198
* Help people contribute councillor names in the format we need them #1199
* Help people who want to contribute councillor information get to the council website where they can find it #1200

**Comment by equivalentideas** [comment]
We've now got instructions on the page that tell you where to get the information from so I'll mark this done.

![screen shot 2017-08-24 at 4 57 04 pm](https://user-images.githubusercontent.com/1239550/29653486-557fd26c-88ed-11e7-88a3-03f37a755542.png)

It would be pretty cool if we could turn that text into a link https://github.com/openaustralia/planningalerts/issues/1200 ✨
